frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -108,6 +108,27 @@ function ReaderRolling:init()
 end
 
 function ReaderRolling:onReadSettings(config)
+    -- 20180503: some fix in crengine has changed the way the DOM is built,
+    -- and may make XPATHs obtained from previous version invalid.
+    -- We may request the previous (buggy) behaviour though, which we do
+    -- if we use a DocSetting previously made that may contain bookmarks
+    -- and highlights with old XPATHs.
+    if not config:readSetting("cre_dom_version") then
+        -- Not previously set, guess which DOM version to use
+        if config:readSetting("last_xpointer") then
```

Comment:
It's also the last page the user will jump to.
But I just wanted something generic and old, something actually just enough to check if it's an empty-just-created sidecar file or not.
It will be a no-op (doesnt matter what cre_dom_version is for epub, the same DOM code will be used).

Reply:
Fair enough, let's keep it that way. I'm just trying to say that for just last_xpointer the trade-off might be worth it.